tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_positive/run.py

```diff
@@ -0,0 +1,77 @@
+from pysys.basetest import BaseTest
```

Comment:
Do we need one such test for az mapper as well?

Reply:
I added here two positive test cases for az. Maybe I didn't get what you mean.

Reply:
Better to have them. Lets do that when the mapper testbed is created, there it should be easier to achieve.